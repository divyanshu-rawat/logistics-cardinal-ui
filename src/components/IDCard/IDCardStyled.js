import styled from 'styled-components';

const IDCardStyled = styled('div')`
  display: flex;
  flex-flow: row;
  align-items: center;
  font-weight: ${({ theme }) => theme.fontWeight.light};
`;

export default IDCardStyled;
