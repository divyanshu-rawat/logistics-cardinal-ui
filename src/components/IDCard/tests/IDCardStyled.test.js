import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import IDCardStyled from '../IDCardStyled';

const renderedComponent = () =>
  shallow(<IDCardStyled theme={{ fontSize: {}, fontWeight: {} }} />);

describe('IDCardStyled', () => {
  it('should render the component with default size', () => {
    const renderComponent = renderedComponent();
    expect(toJson(renderComponent)).toMatchSnapshot();
  });
});
