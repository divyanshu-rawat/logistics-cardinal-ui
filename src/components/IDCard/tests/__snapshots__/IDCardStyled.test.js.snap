// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IDCardStyled should render the component with default size 1`] = `
<div
  className="IDCardStyled-jhufwa ijmGkD"
/>
`;
